src/lang/po/Prusa-Firmware-Buddy.pot

```diff
@@ -699,45 +620,46 @@ msgstr ""
 msgid "PREHEAT for UNLOAD"
 msgstr ""
 
-#: src/gui/screen_printing.cpp:563
+#: src/gui/screen_printing.hpp:46 src/gui/screen_printing.cpp:357
 msgid "PRINTING"
 msgstr ""
 
-#: src/gui/dialogs/DialogFactory.cpp:14 src/gui/screen_menu_filament.cpp:98
+#: src/gui/dialogs/DialogFactory.cpp:15 src/gui/screen_menu_filament.cpp:102
 msgid "PURGE FILAMENT"
 msgstr ""
 
-#: src/gui/dialogs/dialog_response.cpp:19
+#. Ok
+#: src/gui/dialogs/dialog_response.cpp:21
 msgid "PURGE MORE"
 msgstr ""
 
-#: src/gui/dialogs/DialogG162.cpp:27 src/gui/dialogs/DialogLoadUnload.cpp:40
+#: src/gui/dialogs/DialogG162.cpp:9 src/gui/dialogs/DialogLoadUnload.cpp:15
 msgid "Parking"
 msgstr ""
 
-#: src/gui/screen_printing_serial.cpp:35 src/gui/screen_printing.cpp:76
+#: src/gui/ScreenPrintingModel.hpp:16 src/gui/screen_printing.cpp:38
 msgid "Pause"
 msgstr ""
 
-#: src/gui/screen_printing.cpp:77
+#: src/gui/screen_printing.cpp:39
 msgid "Pausing..."
 msgstr ""
 
-#: src/gui/wizard/screen_wizard.cpp:330
+#: src/gui/wizard/xyzcalib.cpp:66 src/gui/wizard/screen_wizard.cpp:102
 msgid ""
 "Place a sheet of paper under the nozzle during the calibration of first 4 "
 "points. If the nozzle catches the paper, power off printer immediately!"
 msgstr ""
 
-#: src/gui/wizard/screen_wizard.cpp:305
+#: src/gui/wizard/xyzcalib.cpp:40 src/gui/wizard/screen_wizard.cpp:77
 msgid "Please clean the nozzle for calibration. Click NEXT when done."
 msgstr ""
 
-#: src/gui/screen_menu_lan_settings.cpp:290
+#: src/gui/screen_menu_lan_settings.cpp:280
 msgid "Please insert a USB drive and try again."
 msgstr ""
 
-#: src/gui/guimain.cpp:197
+#: src/gui/guimain.cpp:187
 msgid ""
 "Please insert the USB\n"
```

Comment:
```suggestion
"Please insert the USB drive that came with the firmware"
```

Reply:
Again - have you seen the actual code? This is a text from an early welcome screen. I'm not touching this.